dt-impact-tracer/src/main/java/edu/washington/cs/dt/impact/plugin1/Sample.java

```diff
@@ -0,0 +1,628 @@
+                args = new String[]{
+                        "-technique", "parallelization",
+                        "-order", "original",
+                        "-origOrder", dtResults + "/" + j + "-order.txt",
+                        "-testInputDir", dtResults + "/sootTestOutput-" + j,
+                        "-filesToDelete", dtResults + "/env-files",
+                        "-project", "",
+                        "-testType", j,
+                        "-numOfMachines", k,
+                        "-outputDir", paraResults,
+                        "-timesToRun", Integer.toString(MEDIANTIMES),
+                        "-classpath", classpath,
+                        "-resolveDependences", precomputeFlag,
+                        postProcessFlag};
+                Runner.nullOutputFileName();
+                TestPluginPlugin.mojo().getLog().info("OneConfigurationRunner Parameters\n\t" + StringUtils.join(args, "\n\t"));
+                OneConfigurationRunner.main(args);
+
+                // Time Order
+                TestPluginPlugin.mojo().getLog().info("Time Order");
+                precomputeFlag =  prioDTLists + "/parallelization-" + "" + "-" + j + "-" + k + "-time.txt";
+                postProcessFlag = "";
+                args = new String[]{
+                        "-technique", "parallelization",
+                        "-order", "time",
+                        "-timeOrder", dtResults + "/" + j + "-time.txt",
+                        "-origOrder", dtResults + "/" + j + "-order.txt",
+                        "-testInputDir", dtResults + "/sootTestOutput-" + j,
+                        "-filesToDelete", dtResults + "/env-files",
+                        "-project", "",
+                        "-testType", j,
+                        "-numOfMachines", k,
+                        "-outputDir", paraResults,
+                        "-timesToRun", Integer.toString(MEDIANTIMES),
+                        "-classpath", classpath,
+                        "-resolveDependences", precomputeFlag,
+                        postProcessFlag};
+                Runner.nullOutputFileName();
+                TestPluginPlugin.mojo().getLog().info("OneConfigurationRunner Parameters\n\t" + StringUtils.join(args, "\n\t"));
+                OneConfigurationRunner.main(args);
+            }
+        }
+
+
+
+        TestPluginPlugin.mojo().getLog().info("Cleaning Up Files");
+        FileUtils.deleteQuietly(new File("tmpfile.txt"));
+        FileUtils.deleteQuietly(new File("tmptestfiles.txt"));
+    }
+
+    /**
+     * This function builds a classpath from the passed string
+     * From https://stackoverflow.com/questions/4752817/expand-environment-variables-in-text
+     *
+     * @param text environment variable
+     * @return returns the complete classpath with the environment_variable expanded
+     */
+    public static String expandEnvVars(String text) {
+        Map<String, String> envMap = System.getenv();
```

Comment:
Minor: This method is in both plugins, should probably be extracted somehow (if it's even needed).